Source/WebCore/dom/MouseEvent.h

```diff
@@ -85,6 +85,8 @@ class MouseEvent : public MouseRelatedEvent {
     WEBCORE_EXPORT virtual RefPtr<Node> fromElement() const;
 
     static bool canTriggerActivationBehavior(const Event&);
+    static bool typeIsClickOrAuxClick(const Event&);
+    static bool typeIsClickOrAuxClick(const AtomString&);
```

Comment:
Maybe a slightly more terse name. I like `isAnyClick`, `isEitherClick`, or `isClickOrAuxClick`.

Maybe these should be non-member functions? I don’t see why we have to make them members of `MouseEvent`. It’s good to have them in this header file, though!